app/views/user_my_modules/_index_edit.html.erb

```diff
@@ -32,7 +32,7 @@
   <% end %>
 <% end %>
 
-<% if can_add_user_to_module(@my_module) and @unassigned_users.count > 0 %>
+<% if manage_users_in_module?(@my_module) and @unassigned_users.count > 0 %>
```

Comment:
use &&

Reply:
We really would need the Hound to bark on the view templates...